pkg/encoding/v1/encoding_test.go

```diff
@@ -21,38 +21,40 @@ func UriPathAddrFromString(s string) *PathRegex {
 
 func TestPortMapping_UnmarshalYAML(t *testing.T) {
 	tests := []struct {
+		Name    string
 		Succeed bool
 		RawPort string
 		Port    *PortMapping
 	}{
-		{true, "5000", &PortMapping{ContainerPort: 5000, ServicePort: 5000}},
-		{true, "5000:80", &PortMapping{ContainerPort: 5000, ServicePort: 80}},
-		{true, "", &PortMapping{}}, // UnmarshalYAML won't be even called for empty strings
-		{false, "x5000", nil},
-		{false, "5000:", nil},
-		{false, "x5000:", nil},
-		{false, ":80", nil},
-		{false, ":80x", nil},
-		{false, "x:80x", nil},
-		{false, "x:80", nil},
-		{false, ":8080:80", nil},
-		{false, "x:8080:80", nil},
-		{false, "x:x8080:x80", nil},
-		{false, ":8080:x80", nil},
-		{false, ":8080:80:", nil},
-		{false, "8080:80:", nil},
-		{false, "8080:80::", nil},
-		{false, ":", nil},
-		{false, "::", nil},
-		{false, ":::", nil},
-		{false, "::::", nil},
-		{false, ":::::", nil},
-		{false, "::1:80", nil},
-		{false, "::1:8080:80", nil},
-		{false, "::1:8080:80:", nil},
+		{"Only ContainerPort", true, "5000", &PortMapping{ContainerPort: 5000, ServicePort: 5000}},
+		{"ContainerPort:ServicePort", true, "5000:80", &PortMapping{ContainerPort: 5000, ServicePort: 80}},
+		{"Empty Portmapping", true, "", &PortMapping{}}, // UnmarshalYAML won't be even called for empty strings
+		{"Failed Portmapping case-1", false, "x5000", nil},
```

Comment:
Instead of using `case-1`, `case-2`, etc, can we put the `RawPort` in there? Like `Failed Portmapping - x5000`? WDYT?